Gemfile

```diff
@@ -57,7 +57,10 @@ gem "csv"
 gem "logger"
 gem "ostruct"
 
+# Gem for shared code across engines
 gem "bops_core", path: "engines/bops_core"
+
+gem "bops_consultees", path: "engines/bops_consultees"
 gem "bops_admin", path: "engines/bops_admin"
 gem "bops_api", path: "engines/bops_api"
 gem "bops_config", path: "engines/bops_config"
```

Comment:
```suggestion
gem "bops_admin", path: "engines/bops_admin"
gem "bops_api", path: "engines/bops_api"
gem "bops_config", path: "engines/bops_config"
gem "bops_consultees", path: "engines/bops_consultees"
```

Reply:
Ugh clearly not paying attention today, thanks/